src/context/AuthContext.tsx

```diff
@@ -11,11 +11,10 @@ export const AuthContext = createContext({} as IAuthContext)
 
 export const AuthProvider = ({ children }: { children: ReactNode }) => {
   const token = GetToken()
-
   const [isAuth, setIsAuth] = useState(token ? true : false)
 
   useEffect(() => {
-    token ? setIsAuth(true) : localStorage.removeItem('login')
+    token ? setIsAuth(true) : setIsAuth(false)
```

Comment:
remove token and logout user?

Reply:
Why should i try to remove the token if GetToken tries to get it from localStorage? I should only remove it when implementing logout, no?

Reply:
Yeah, seems good